openehr-rm/src/main/java/com/nedap/archie/rm/datastructures/ItemSingle.java

```diff
@@ -15,20 +15,20 @@
 @XmlType(name = "ITEM_SINGLE", propOrder = {
         "item"
 })
-public class ItemSingle extends ItemStructure<Item> {
+public class ItemSingle extends ItemStructure<Element> {
 
-    private Item item;
+    private Element item;
```

Comment:
TODO: maybe postpone to major release?

Reply:
I'll keep this, just a bug fix anyway, if people were using a cluster they have major problems :)